ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir/: "local_backups"
```

Comment:
I'm assuming that the directory name supplied does not have a trailing slash, so it is needed to form the path